### PR TITLE
"fix" for backslash in message id

### PR DESCRIPTION
**Pull request by DavidBoike**

### CODE REVIEW

File: src/ServiceControl.UnitTests/BodyStorage/RavenAttachmentsBodyStorageTests.cs [hunk 1]
```diff
@@ -23,6 +23,12 @@ public async Task Attachments_with_ids_that_contain_backslash_should_be_readable
 
                 var retrieved = await bodyStore.TryFetch(messageId);
                 Assert.True(retrieved.HasResult);
+                Assert.AreEqual(contentType, retrieved.ContentType);
```
Comment (danielmarbach) [inline src/ServiceControl.UnitTests/BodyStorage/RavenAttachmentsBodyStorageTests.cs:26]: Should this be reset?

File: src/ServiceControl/CompositeViews/Messages/GetMessagesController.cs [hunk 2]
```diff
@@ -1,15 +1,24 @@
 namespace ServiceControl.CompositeViews.Messages
 {
+    using ServiceControl.Operations.BodyStorage.Api;
     using System.Net.Http;
     using System.Threading.Tasks;
     using System.Web.Http;
 
     public class GetMessagesController : ApiController
```
Comment (danielmarbach) [inline src/ServiceControl/CompositeViews/Messages/GetMessagesController.cs:8]: Should we mention in a comment that messages routes need to be on this controller to not mess up catch all?

File: src/ServiceControl.Audit/Auditing/MessagesView/GetMessagesController.cs [hunk 3]
```diff
@@ -6,10 +6,17 @@ namespace ServiceControl.Audit.Auditing.MessagesView
 
     public class GetMessagesController : ApiController
```
Comment (danielmarbach) [inline src/ServiceControl.Audit/Auditing/MessagesView/GetMessagesController.cs:7]: Should we mention in a comment that messages routes need to be on this controller to not mess up catch all?

Comment (DavidBoike) [reply src/ServiceControl.Audit/Auditing/MessagesView/GetMessagesController.cs:9]: Added comment

File: src/ServiceControl.UnitTests/BodyStorage/RavenAttachmentsBodyStorageTests.cs (hunk 1 above)
Comment (DavidBoike) [reply src/ServiceControl.UnitTests/BodyStorage/RavenAttachmentsBodyStorageTests.cs:26]: Keeping - this makes the test more explicit, ensuring that this bug won't regress at the storage level.

File: src/ServiceControl/CompositeViews/Messages/GetMessagesController.cs (hunk 2 above)
Comment (DavidBoike) [reply src/ServiceControl/CompositeViews/Messages/GetMessagesController.cs:10]: Added comment

**Comment by DavidBoike** [comment]
Ready for review

**Pull request merged**